packages/client/src/components/atoms/DateSwitcher/DateSwitcher.tsx

```diff
@@ -41,7 +41,7 @@ const DateSwitcher: React.FC<Props> = ({ currentDate, jump, ...MenuProps }) => {
       <Menu data-testid={__calendarMenuId__} {...MenuProps}>
         <CalendarPicker
           date={currentDate}
-          onChange={(currentDate) => {
+          onChange={(currentDate: DateTime | null) => {
```

Comment:
What's happening here?

Reply:
eslint was complaining

Reply:
Ok, it's weird that this should solve it...can you fix it down below: not use the `!` but preform a check if it's truthy (or whatever) so that when you're calling `changeCalendarDate()` you're passing the right type...that's the whole point of TS.